app/src/main/java/com/example/leagueoflegends/model/Response.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.leagueoflegends.model
+
+data class Response(val data: Map<String, Champion>) {
+    data class Champion(val id: String, val name: String, val title: String, val blurb: String, val image: Image) {
```

Comment:
I'm quite impressed that this worked. I haven't tried putting these data classes into each other.
Did it work?